dev/com.ibm.ws.javaee.ddmodel/test/com/ibm/ws/javaee/ddmodel/web/WebAppTest.java

```diff
@@ -177,167 +180,363 @@ public void testEE6Web30DenyUncoveredHttpMethods() throws Exception {
+                                                               "<secure>true</secure>" +
+
+                                                               "<attribute>" +
+                                                               "<description xml:lang=\"en\">Hair color</description>" +
+                                                               "<description xml:lang=\"fr\">Couleur de cheveux</description>" +
+                                                               "<attribute-name>color</attribute-name>" +
+                                                               "<attribute-value>blue</attribute-value>" +
+                                                               "</attribute>" +
+
+                                                               "<attribute>" +
+                                                               "<description xml:lang=\"en\">Facial complecxion</description>" +
+                                                               "<description xml:lang=\"fr\">Complexión facial</description>" +
+                                                               "<attribute-name>complexion</attribute-name>" +
+                                                               "<attribute-value>rough</attribute-value>" +
+                                                               "</attribute>" +
+
+                                                               "</cookie-config>" +
+                                                               "</session-config>"),
+                                    WebApp.VERSION_6_0);
+
+        CookieConfig cookieConfig = webApp.getSessionConfig().getCookieConfig();
+        Assert.assertNotNull(cookieConfig);
+
+        List<AttributeValue> attributes = cookieConfig.getAttributes();
+
+        // First Attribute
+        AttributeValue attribute = attributes.get(0);
+        List<Description> descriptions = attribute.getDescriptions();
+
+        //     first description
+        Description description = descriptions.get(0);
+        Assert.assertEquals(description.getLang(), "en");
+        Assert.assertEquals(description.getValue(), "Hair color");
+
+        //    second description
+        description = descriptions.get(1);
+        Assert.assertEquals(description.getLang(), "fr");
+        Assert.assertEquals(description.getValue(), "Couleur de cheveux");
+
+        // Second Attribute
+        attribute = attributes.get(1);
+        descriptions = attribute.getDescriptions();
+
+        //     first description
+        description = descriptions.get(0);
+        Assert.assertEquals(description.getLang(), "en");
+        Assert.assertEquals(description.getValue(), "Facial complecxion");
+
+        //     second description
+        description = descriptions.get(1);
+        Assert.assertEquals(description.getLang(), "fr");
+        Assert.assertEquals(description.getValue(), "Complexión facial");
```

Comment:
Same here...should attribute-name and value pair be tested?

Reply:
Added tests on attribute-name.